### DISCUSSION TITLE
Alerts - Does this plugin support alerts when i got a specific string response?

### DISCUSSION THREAD

**Discussion by Santiagososs** [grafana/grafana-infinity-datasource#448] [Q&A]
Can i get a alert when i got an specific response from a json ?

Currently i have a Rancher running some aplications and using the Infinity plugim i get the name and state information, so i can clearly see if any of the apps are down.

But the idea its that when a app goes down, grafana send me an alert via Telegram

So when i get the State response "stopped" grafana gonna send me a alert, is that possible using Infinity Plugim ?

Plugim Version: **1.1.0**

Dashboard config: 
![image](https://user-images.githubusercontent.com/70164638/200629823-110bd536-1f6d-43eb-81ed-75f58cc6eb90.png)

**Comment by yesoreyeram** [comment]
yes. alerts are possible. Use "Backend" as parser and setup alert on top of it like any other grafana alert.

**Reply by Santiagososs** [reply]
I'm now using Backend as parser and formated my alert to get my two informations, Name and State, but keep I still can't create or configure alerts, if i change the data visualization to time series for example i get a No Data, and in Grafana Alerts Tab i have a mensage telling-me that i have no compatible data sources, how exactly i do that ? 

My Infinity Query
![image](https://user-images.githubusercontent.com/70164638/200645378-55cc06aa-bbd4-4579-b326-6aae7db0cc18.png)

My Alerts Tab
![image](https://user-images.githubusercontent.com/70164638/200645530-899b323f-c22f-4835-9e8a-dd26fc04ee3c.png)

**Reply by yesoreyeram** [reply]
Probably you may need to restart grafana after installing the plugin.. 🤔 not sure what's going on with your setup. Give a try

**Comment by Santiagososs** [comment]
Att- The alert bar still doesn't recognize infinity, any troubleshooting ideas?

**Comment by Passe12345** [comment]
Grafana alerting needs interger to work on an Expression. I couldn't figure out if it's possible to use the Computed columns or Filter to convert the result of a string Column to a boolean which can then be evaluated as an expression in the alert rule creation.

Example, I have 2 columns:
- conclusion: 2 possible values: `success` or `failure`
- created_at: a Timestamp (Time Format Default ISO)

@yesoreyeram, is it possible?

**Reply by yesoreyeram** [reply]
it should be already doable with computed columns. `state == 'success' ? 200 : 500`

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/153843/202755475-db6e047d-3129-45db-a022-d793b0dc36b9.png">

but more options coming in 1.2.0 via JSONata in backend root selector. You can try [this](https://github.com/yesoreyeram/grafana-infinity-datasource/releases/tag/v1.2.0) here.
